storage/connect/mysql-test/connect/t/odbc_postgresql.test

```diff
@@ -205,3 +205,9 @@ CREATE VIEW v1 AS SELECT * FROM t1;
 SELECT * FROM v1;
 DROP VIEW v1;
 DROP TABLE t1;
+
+# MDEV-25767 DELETE with WHERE condition crashes when two columns used for
```

Comment:
Can we reuse here `schema.t2` instead of creating a new schema in `.sql` file?

Reply:
> Can we reuse here `schema.t2` instead of creating a new schema in `.sql` file?

Only if we change schema.t2. It two a minimum columns to trigger. All the test ones there are single column.

Reply:
I would suggest to update comment section at the beginning
from:
```
$ psql -U postgres < storage/connect/mysql-test/connect/t/odbc_postgresql.sql 
psql: error: FATAL:  Peer authentication failed for user "postgres"
```
to:
```
$ sudo -u postgres psql < storage/connect/mysql-test/connect/t/odbc_postgresql.sql
```

Reply:
> I would suggest to update comment section at the beginning from:

That is definitely out of scope for this PR and there are quite a few things that need updating there.

> Do we need 5. step `allow user mtr...` ?

Yes, otherwise mtr can't connect to Postgres. Although there are better ways, but as above, out of scope.

> Also to rebase to 10.3 as earliest affected branch.

OK

> Commit message is also referencing the UPDATE statement,so would be good to add that case too.

Due to a completely different bug I found (MDEV-29397) I can't actually do that right now. Test cases will come when I fix that one.